### PR TITLE
Update clean_name to handle pathlib.Path names.

### PR DESCRIPTION
**Pull request by m000**

### CODE REVIEW

**Comment by jschneier** [comment]
Thanks, can you add a test?

**Comment by jschneier** [comment]
Also needs a rebase.

File: storages/utils.py [hunk 1]
```diff
@@ -32,6 +33,10 @@ def clean_name(name):
     """
     Cleans the name so that Windows style paths work
     """
+    # Convert pathlib.Path to string.
+    if isinstance(name, Path):
+        name = str(Path)
```
Comment (jschneier) [inline storages/utils.py:38]: Also this should be `str(name)`.

**Comment by jschneier** [comment]
Thanks for the patch, I've cleaned this up and will merge in #1200.

**Comment by m000** [comment]
@jschneier Thanks! Sorry I couldn't pick it up again on time.

**Pull request closed without merging**